lib/simple_config.py

```diff
@@ -367,7 +367,7 @@ def get_fee_text(self, pos, dyn, mempool, fee_rate):
         text is what we target: static fee / num blocks to confirm in / mempool depth
         tooltip is the corresponding estimate (e.g. num blocks for a static fee)
         """
-        rate_str = (format_satoshis(fee_rate/1000, False, 0, 0, False)  + ' sat/byte') if fee_rate is not None else 'unknown'
+        rate_str = format_satoshis(fee_rate/1000, 0, 0, precision=FEE_PRECISION) + ' sat/byte' if fee_rate is not None else 'unknown'
```

Comment:
I think we should replace the inline `if` with a multi-line standard `if`.
It always takes me a few seconds to think about operator precedence when seeing long expressions without brackets.. :) but also, the line is getting long, so let's do standard `if`

Reply:
Yeah. I never liked python's "ternary", given it buries the lede.